### DISCUSSION TITLE
jtlwrite default timestamp

### DISCUSSION THREAD

**Discussion by timothy117** [abstracta/jmeter-java-dsl#180] [General] [answered]
default .jtl from jmeter given `#jmeter.save.saveservice.timestamp_format=yyyy/MM/dd HH:mm:ss.SSSS’`
but currently we are using epoch timestamp. anywhere we can change that to make it human readable ?

**Comment by rabelenda** [comment, marked as answer]
Hello, thank you for asking!

Have you tried using JMeter properties as described [here](https://abstracta.github.io/jmeter-java-dsl/guide/#properties) for setting the `jmeter.save.saveservice.timestamp_format` property?

Can you give more info, maybe some example. Because you say you are using epoch but then ask to make it human readable? Are  you refering to jtlWriter service storing timestamps as epoch in jtl files instead of using the setting specified by that system property?

**Reply by rabelenda** [reply]
@timothy117 have you had a chance to review my comment?
